openclaw_hooks/nima-recall-live/lazy_recall.py

```
@@ -1,468 +1,438 @@
+    if fts_candidates and len(fts_candidates) >= FTS_MIN_RESULTS:
+        best_fts_score = min(c.get('fts_score', 999) for c in fts_candidates)
+        if best_fts_score < FTS_CONFIDENCE_THRESHOLD:
+            skipped_embedding = True
+            print(f"[lazy_recall] Skipping embedding (FTS score {best_fts_score:.1f})", 
+                  file=sys.stderr)
+    
+    # 4. Embedding search (if needed)
+    emb_candidates = []
+    if use_embedding and not skipped_embedding:
+        # Try pre-computed index first
+        matrix, meta = load_embedding_index()
```

Ruff (0.15.0)

[warning] Unpacked variable `meta` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)